src/App.js

```diff
@@ -4,15 +4,25 @@ import { Routes, Route, RouterProvider } from "react-router-dom";
 import { LoginPage } from "./components/LoginPage.jsx";
 import { HomePage } from "./components/HomePage.jsx";
 import { IndvReview } from "./components/IndvReview.jsx";
-import { useState, useEffect } from "react";
-import { getReviews } from "./api/api.js";
+import { UserContext } from "./contexts/UserContext.js";
+import { useEffect, useState, useContext } from "react";
+import { getUsers } from "./api/api";
 
 function App() {
+  const [users, setUsers] = useState([]);
+  const userValueFromContext = useContext(UserContext);
+
+  useEffect(() => {
+    getUsers().then((users) => {
+      setUsers(users);
+    });
+  }, []);
   return (
     <div className="App">
       <Header />
 
       <Routes>
+        <Route path="/" element={<LoginPage users={users} />} />
```

Comment:
This does work great, but if I log in as someone other than tickle, once I refresh the page on the homepage, I get logged in as tickle. It would be better if you don't have a default logged in user to begin with and then have a special login page where you can log in as your chosen user